11.curiosity.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.            # Forward prediction: predict next state feature, given current state feature and action (one-hot)
</pre><p> </p><p>Suggestion (Google Docstyle):</p><p> </p><p> </p><pre class="ql-syntax" spellcheck="false">"""Forward prediction.

predict next state feature, given current state feature and action (one-hot).
"""
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Curt-Park/rainbow-is-all-you-need/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>